components/service/firefox-accounts/README.md

```diff
@@ -173,7 +173,7 @@ val accountEventsObserver = object : AccountEventsObserver {
     override fun onEvents(event: List<AccountEvent>) {
         // device received some commands; for example, here's how you can process incoming Send Tab commands:
         commands
-            .filter { it is AccountEvent.IncomingDeviceCommand }
+            .filter { it is AccountEvent.CommandReceived }
```

Comment:
I had to change the name of this variant for dumb code-gen-related reasons, but I also prefer "command received" because it maps more directly to the of the push notification.